File: 1. Fundamentos de programación/Prácticas/Práctica 2, Programas simples/Java/ParteII_Seleccion_Compara_Numeros/ParteII_Seleccion_Compara_Numeros.java
/* [Felicidades, está muy bien comentado el código, muy bien estructurado,
 los nombres de las variables son explicativas y el programa corre conforme a lo esperado.
 Gracias, me has estado ayudando mucho. :3
 
 Si acaso sería cambiar el nombre del archivo y ponerle un identificador con base al pdf, p.e:
    ParteII_Seleccion_Compara_Numeros.java
 ¡RODRIGO!]
 */



/*
    ALGORITMO: [Thanks...]
    1. El programa muestra la carátula.
    2. El programa solicita al usuario escribir un número entero.
    3. El usuario ingresa un número entero.
    4. El programa solicita al usuario escribir otro número entero.
    5. El usuario ingresa un número entero.
    6. El programa compara ambos número enteros.
    7. El programa muestra si los números son iguales o si uno es mayor que otro.
    8. El programa muestra Fin del programa.
    9. Fin del programa. 
*/

/*
 * Rubén Ramírez Cervantes
 * 2203007463
 * bicinrubenbr@gmail.com
 *
 * Compilar: javac ParteII_Seleccion_Compara_Numeros.java
 * Ejecutar: java ParteII_Seleccion_Compara_Numeros [OK]
 */

//Clase ParteII_Seleccion_Compara_Numeros
public class ParteII_Seleccion_Compara_Numeros{

    //Función main()
	public static void main(String args[]){

        //Declaración e importación.
        java.util.Scanner leerInt = new java.util.Scanner(System.in); 
        // [Esta es una forma de llamar al objeto Scanner y es válida ¡RODRIGO!]
        // La otra es utilizar: import java.util.Scanner;


        //Se muestra Carátula.
        muestraCaratula();

        //Lectura de datos int
        System.out.println("Ingresa tu primer número:");
        int numeroAIn = leerInt.nextInt();

        System.out.println("Ingresa tu segundo número:");
        int numeroBIn = leerInt.nextInt();

        //Comparaciones de los datos ingresados
        if (numeroAIn == numeroBIn) {
            System.out.println("El primer número y el segundo número son iguales: " + numeroAIn + " = " + numeroBIn);
        }

        else if(numeroAIn > numeroBIn){
            System.out.println("El número " + numeroAIn + " es mayor que " + numeroBIn);
        }

        else{
            System.out.println("El número " + numeroBIn + " es mayor que " + numeroAIn);
        }

        //Se muestra Fin del programa.
        finPrograma();
    }//Fin función main()

    //Método Carátula.
    public static void muestraCaratula(){
        System.out.println("\t**************************************");
        System.out.println("\t*   FUNDAMENTOS DE LA PROGRAMACION   *");
        System.out.println("\t*   ALUMNO: Rubén Ramírez Cervantes  *");
        System.out.println("\t*        MATRICULA: 2203007463       *");
        System.out.println("\t*          COMPARAR NUMEROS          *");
        System.out.println("\t**************************************\n");
    }//Fin método muestraCaratula()

    //Método Fin del programa.
    public static void finPrograma(){
        System.out.println("\n\n*** Fin del programa. ***\n");
    }//Fin método finPrograma()
}//Fin Clase ParteII_Seleccion_Compara_Numeros{}

File: 1. Fundamentos de programación/Practicas/Práctica 2, Programas simples/C/ParteII_Seleccion_Compara_Numeros/ParteII_Seleccion_Compara_Numeros.c
/*
    ALGORITMO:
    1. El programa muestra la carátula.
    2. El programa solicita al usuario escribir un número entero.
    3. El usuario ingresa un número entero.
    4. El programa solicita al usuario escribir otro número entero.
    5. El usuario ingresa un número entero.
    6. El programa compara ambos número enteros.
    7. El programa muestra si los números son iguales o si uno es mayor que otro.
    8. El programa muestra Fin del programa.
    9. Fin del programa. 
*/

/*
 * Rubén Ramírez Cervantes
 * 2203007463
 * bicinrubenbr@gmail.com
 *
 * Compilar: gcc ParteII_Seleccion_Compara_Numeros.c -o ParteII_Seleccion_Compara_Numeros.out
 * Ejecutar: ./ParteII_Seleccion_Compara_Numeros.out
 */

//Importaciones
#include <stdio.h>
//#include <iostream>
#include <stdlib.h>

//Método ParteII_Seleccion_Compara_Numeros
int main(){
	int numeroAIn, int numeroBIn;

	muestraCaratula();

	//Lectura de datos int
    printf("Ingresa tu primer número: ");
    scanf("%i", & numeroAIn);

    printf("\ntus numero son: %i", numeroAIn);

    //printf("Ingresa tu segundo número: ");
    //scanf("%i", & numeroBIn);

    /*/Comparaciones de los datos ingresados
    if (numeroAIn == numeroBIn) {
        printf("El primer número y el segundo número son iguales: " + numeroAIn + " = " + numeroBIn);
    }

    else if(numeroAIn > numeroBIn){
        printf("El número " + numeroAIn + " es mayor que " + numeroBIn);
    }

    else{
        printf("El número " + numeroBIn + " es mayor que " + numeroAIn);
    }*/

    //Se muestra Fin del programa.
    finPrograma();

	return 0;
}//Fin Método ParteII_Seleccion_Compara_Numeros{}

//Método Carátula.
static void muestraCaratula(){
    printf("\t**************************************\n");
    printf("\t*   FUNDAMENTOS DE LA PROGRAMACION   *\n");
    printf("\t*   ALUMNO: Rubén Ramírez Cervantes  *\n");
    printf("\t*        MATRICULA: 2203007463       *\n");
    printf("\t*          COMPARAR NUMEROS          *\n");
    printf("\t**************************************\n");
}//Fin método muestraCaratula()

//Método Fin del programa.
static void finPrograma(){
    printf("\n\n*** Fin del programa. ***\n");
}//Fin método finPrograma()

//INVESTIGAR POR QUÉ NO FUNCIONA
    //scanf("%d", numeroAIn);
    //printf("%d", numeroAIn);